docs/en/enterprise-edition/content-collections/administration/setup-sso-integration-on-prisma-cloud/get-started-with-oidc-sso/get-started-with-oidc-jit.adoc

```diff
@@ -0,0 +1,43 @@
+:topic_type: task
+[.task]
+== Get Started with OIDC JIT
+
+Prisma Cloud offers System Administrators the ability to auto-provision users using the Open ID Connect (OIDC) Single Sign-On (SSO) configuration. This option allows you to manage user identities outside of Prisma Cloud in your organization’s Identity Provider (IdP). Configure OIDC Just in Time (JIT) provisioning to create new Prisma Cloud users on an as needed basis, when users with the appropriate entitlements log in using OIDC SSO. 
+
+Complete the steps below on the Prisma Cloud console and your IdP to set up OIDC JIT provisioning:
+
+[.procedure]
+. Log in to Prisma Cloud using an account with System Administrator privileges to configure SSO.
```

Comment:
I think this level isn't needed. I think it would be better to avoid so much nesting in the proceedure.